payloads_test.go

```diff
@@ -1745,6 +1745,92 @@ const listDomainsPayload = `{
   ]
 }`
 
+const listSharedDomainsPayload = `{
+  "total_results": 6,
```

Comment:
Can you please reduce the number of results? the `payloads_test.go` is getting bigger and bigger, so let's add just the tests that we really need.